README.md

```diff
@@ -56,6 +56,43 @@ each system.
 
 The Visual Studio Code extension for ROS supports launch debugging for ROS 1 and ROS 2 nodes. The VSCode extension currently supports debugging ROS written in Python and C++. The ROS node or nodes to be debugged must be placed in a ROS launch file with the extension `.launch` for ROS1 or ROS2 or with the extension `.py` for ROS2.
 
+### Starting with no launch.json
+`vscode/launch.json` is a file which defines a debug launch configuration within VSCode. To create a `launch.json` with ROS debugging support -  First, C++ or Python file is selected, vscode uses the selected file to seed the launch creation UI. Next click the `Run and Debug` tab on the left sidebar, then select the link to create a `launch.json` file. VSCode will drop down from the command pallet with a list of options, which includes 'ROS'. Select this option. 
```

Comment:
[nit] the tense changes here. This might be clearer as a bullet or numbered list?